fe/fe-core/src/main/java/org/apache/doris/statistics/util/StatisticsUtil.java

```diff
@@ -177,7 +177,7 @@ public static AutoCloseConnectContext buildConnectContext(boolean limitScan) {
         sessionVariable.enablePageCache = false;
         sessionVariable.parallelExecInstanceNum = Config.statistics_sql_parallel_exec_instance_num;
         sessionVariable.parallelPipelineTaskNum = Config.statistics_sql_parallel_exec_instance_num;
-        sessionVariable.setEnableNereidsPlanner(false);
+        sessionVariable.setEnableNereidsPlanner(true);
```

Comment:
Since  we are going to use nereids planner, better set forbidUnknownColStats = false, otherwise the collection may fail when there is no stats for the collecting table.

Reply:
forbidUnknownColStats is disabled by default, but you are right,we should set explicitly here